### PR TITLE
Designing Profile Update Interface

### PR DESCRIPTION
**Pull request by shawon-sharma**

### CODE REVIEW

File: app/src/main/java/com/framgia/elsytem/ProfileActivity.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+
+
+
+    }
+
+}
+class  ViewAdapter extends ArrayAdapter<String>
+{
+    Context c;
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/elsytem/ProfileActivity.java]: re-format this file please!

File: app/src/main/res/layout/content_profile.xml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+            <ImageView
+                android:id="@+id/img"
+                android:layout_width="@dimen/img_layout_width"
+                android:layout_height="@dimen/img_layout_height"
+                android:src="@drawable/ic_person_outline_black_36dp"/>
+            <TextView
+                android:id="@+id/person"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="shawon sharma"
```
Comment (nvmanh) [inline app/src/main/res/layout/content_profile.xml]: do not hardcode like this

File: app/src/main/res/layout/profile_list.xml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        android:layout_height="wrap_content"
+        android:src="@drawable/e"
+        android:id="@+id/imageView"
+        android:layout_marginRight="5dp"/>
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textAppearance="?android:attr/textAppearanceLarge"
+        android:text="Large Text"
```
Comment (nvmanh) [inline app/src/main/res/layout/profile_list.xml]: hard code

File: app/src/main/res/layout/profile_list.xml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        android:layout_toRightOf="@+id/imageView"
+        android:layout_alignParentRight="true"
+        android:layout_alignParentEnd="true"
+        />
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textAppearance="?android:attr/textAppearanceMedium"
+        android:text="Medium Text"
```
Comment (nvmanh) [inline app/src/main/res/layout/profile_list.xml]: hard code

File: app/src/main/res/layout/content_profile.xml (hunk 2 above)
Comment (shawon-sharma) [inline app/src/main/res/layout/content_profile.xml]: these are the sample text thats why i make its hardcoded

**Comment by nvmanh** [comment]
i know that your sample text, but in case user do not input this value. In view we will see this, so no sample text hard code is permitted!

**Pull request merged**